### DISCUSSION TITLE
bigfix/9.5/platform/Platform/Config/c_changing_the_client_icon

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#82] [Q&A]
# bigfix/9.5/platform/Platform/Config/c_changing_the_client_icon

By default, the icon in the upper left corner of the client UI is the BigFix logo.

https://help.hcl-software.com/bigfix/9.5/platform/Platform/Config/c_changing_the_client_icon.html

**Comment by kb5ylg** [comment]
What size attributes should the icon be?
